src/mod/databases/mod_mariadb/mod_mariadb.c

```diff
@@ -478,6 +480,9 @@ switch_status_t mariadb_handle_connect(mariadb_handle_t *handle)
 	// https://mariadb.com/kb/en/library/using-the-non-blocking-library/
 	mysql_options(&handle->con, MYSQL_OPT_NONBLOCK, 0);
 
+	// Enable automatic reconnect with the mariadb_reconnect function, without this that function does not work
```

Comment:
Minor, but please use `/* */` comment style

Reply:
ok - was just matching the existing syntax two lines above it.... but happy to change.

Reply:
updated.